bindings/ergo-lib-wasm/tests/test_rest_api.js

```diff
@@ -0,0 +1,13 @@
+import { assert } from 'chai';
+
+import {
+  get_info, NodeConf,
+} from '../pkg/ergo_lib_wasm';
+
+// it('node REST API get_info endpoint', async () => {
+//   let node_conf = new NodeConf("213.239.193.208:9053");
+//   assert(node_conf != null);
+//   let res = await get_info(node_conf);
+//   assert(res != null);
+//   assert(node_conf != null);
+// });
```

Comment:
Can we re-enable the test?

Reply:
Good catch! But no, :) it does not work since we're running these JS test on node in CI and this implementation works only in the browser (`reqwest` implementation for Wasm target use browser's Fetch API). I made https://github.com/ergoplatform/sigma-rust/issues/522 to run JS tests in the browser on CI. When its implemented I hope I remember to uncomment it. :)